packages/addon-dev/src/rollup.ts

```diff
@@ -74,8 +74,9 @@ export class Addon {
   output() {
     return {
       dir: this.#destDir,
-      format: 'es',
       entryFileNames: '[name]',
+      experimentalMinChunkSize: 0,
+      format: 'es',
```

Comment:
I alphabetized the keys. When there is a list of configuration keys, people tend to add a new one to the end of the list. I find unsorted lists not easy to understand and maintain.